### DISCUSSION TITLE
intercept the title content, In order to obtain the result of Specify content

### DISCUSSION THREAD

**Discussion by tyf2018** [SilentVoid13/Templater#463] [Help] [answered]
Title: testAABCCCCCDDDDDFGGG
I want to use commands like <% tp.file.title.split("B")[1] %>, <% tp.file.title.split("F")[0] %> to intercept the title content, In order to obtain the result of CCCCCDDDDD.
Is there a way to achieve it? Thanks!

![2021-12-03_12-18-48](https://user-images.githubusercontent.com/56342788/144544540-c18720dd-ecc4-4646-a3b6-f8c2a0c9e84b.jpg)

**Comment by AB1908** [comment, marked as answer]
Chain the splits? `tp.file.title.split("B")[-1].split("F")[0]`
